docs/getting-started/cash-app-pay.md

```diff
@@ -176,7 +169,7 @@ payKit.createCustomerRequest(request)
 
 ### Step 6A: Add an Authorize Request Event to Cash App Pay button
 
-Once the Cash App PayKit SDK is in the `ReadyToAuthorize` state, you can display the Cash App Pay button. When the Customer taps the button, you can authorize the customer request. Documentation for a Cash App Pay button component can be found [here][cash-button-docs]{:target='_blank'}
+Once the Cash App Pay Kit SDK is in the `ReadyToAuthorize` state, you can display the Cash App Pay button. When the Customer taps the button, you can authorize the customer request. Documentation for a Cash App Pay button component can be found [here][cash-button-docs]{:target='_blank'}
```

Comment:
```suggestion
Once the Cash App Pay Kit SDK is in the `ReadyToAuthorize` state, you can display the Cash App Pay button. When the customer taps the button, you can authorize the customer request. See [Cash Button Docs][cash-button-docs]{:target='_blank'} to learn more about the Cash App Pay button component.
```

Reply:
Please change the last line to " See [[CashPayKitButton][cash-button-docs]{:target='_blank'} to learn more about the Cash App Pay button component."